e2e2/test/cases/nvidia/manifests/efa-device-plugin.yaml

```diff
@@ -52,6 +52,7 @@ spec:
                       - p4de.24xlarge
                       - trn1.32xlarge
                       - trn1n.32xlarge
+                      - p5.48xlarge
```

Comment:
p5 capacity is pretty limited and restricted to the AZ, do we have some control over it so that we can easily tune/stabilize the test?

Reply:
This is adding p5 types to the nodeSelector for the EFA device plugin, it's just a pre-req for using p5's.